templates/expo-template-default/app/(tabs)/_layout.tsx

```diff
@@ -1,35 +1,7 @@
-import { Tabs } from 'expo-router';
 import React from 'react';
```

Comment:
Would recommend using _layout.tsx and _layout.web.tsx as a way to demo this functionality. Also should switch the layouts so it's tabs -> stack instead of stack -> tabs.

Reply:
Since those designs don't have a modal anywhere, we actually don't even need the stack, maybe we just move the tabs to the root navigation level instead? 

Unless you want to demo stack and modal, in which case we should use them somewhere.